File: Data Structures & Algorithms/two-integer-sum/submission-1.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map <Integer,Integer> st =new HashMap<>();
        for(int i=0;i<nums.length;i++){
            int cp= target - nums[i];
            if(st.containsKey(cp)){
                return new int[]{st.get(cp),i};
            }
            st.put(nums[i],i);
        }
        return new int[]{};
    }
}

File: Data Structures & Algorithms/concatenation-of-array/submission-1.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        int n=nums.size();
        vector <int> ans(2*n);
        int i=0;
        for(int x=0;x<n;x++){
            ans[i]=nums[x];
            ans[i+n]=nums[x];
            i++;
        }
        return ans;
    }
};